packages/cli/src/zosfiles/download/uss/UssFile.handler.ts

```diff
@@ -25,13 +25,21 @@ export default class UssFileHandler extends ZosFilesBaseHandler {
             stageName: TaskStage.IN_PROGRESS
         };
         commandParameters.response.progress.startBar({task});
-        return Download.ussFile(session, commandParameters.arguments.ussFileName, {
+
+        const downloadOptions: IDownloadOptions = {
             binary: commandParameters.arguments.binary,
             encoding: commandParameters.arguments.encoding,
             file: commandParameters.arguments.file,
             task,
             responseTimeout: commandParameters.arguments.responseTimeout,
-            overwrite: commandParameters.arguments.overwrite
-        });
+            overwrite: commandParameters.arguments.overwrite,
+        };
+        const attributes = ZosFilesAttributes.loadFromFile(
+            commandParameters.arguments.attributes,
+        );
+        if (attributes != null) {
+            downloadOptions.attributes = attributes;
+        }
+        return Download.ussFile(session, commandParameters.arguments.ussFileName, downloadOptions);
```

Comment:
Should we set `attributes` as part of the initial declaration on line 29, eliminating the need for lines 40-42? 

Assuming that the result of `ZosFilesAttributes.loadFromFile` is falsy, then the attributes option will be ignored in `Download.ussFile`. If an attributes file was successfully loaded, then it will be handled as a valid option in that function.